### PR TITLE
Another shading branch

### PR DESCRIPTION
**Pull request by JaimieGR**
updated hill shading using arcgis planar method

### CODE REVIEW

File: rgen/rgen-viewer/src/main.rs [hunk 1]
```diff
@@ -184,8 +215,8 @@ pub fn main() -> Result<(), String> {
       f.render(0, 0, format!("X: {x:0.2} Z: {z:0.2}", x = hover_pos.x, z = hover_pos.z));
       f.render(0, 24, format!("Height: {meter_height:0.2}"));
 
-      let biome = world.biome_at(hover_pos);
-      f.render(0, 48, format!("Biome: {}", world.context.biomes.name_of(biome)));
+      //let biome = world.biome_at(hover_pos);
+      //f.render(0, 48, format!("Biome: {}", world.context.biomes.name_of(biome)));
```
Comment (macmv) [inline rgen/rgen-viewer/src/main.rs:219]: why'd you comment this out?

File: rgen/rgen-viewer/src/main.rs [hunk 2]
```diff
@@ -268,11 +299,13 @@ impl Render {
     self.canvas.clear();
   }
 
-  pub fn present(&mut self) { self.canvas.present(); }
+  pub fn present(&mut self) {
+    self.canvas.present();
+  }
```
Comment (macmv) [inline rgen/rgen-viewer/src/main.rs:304]: you should change your formatter to the nightly branch

**Review by macmv — COMMENTED** [review]
looks good just one question

File: rgen/rgen-viewer/src/main.rs (hunk 1 above)
Comment (JaimieGR) [reply rgen/rgen-viewer/src/main.rs:219]: It was throwing up an error

File: rgen/rgen-viewer/src/main.rs (hunk 2 above)
Comment (JaimieGR) [reply rgen/rgen-viewer/src/main.rs:304]: could you explain that to me

File: rgen/rgen-viewer/src/main.rs (hunk 1 above)
Comment (macmv) [reply rgen/rgen-viewer/src/main.rs:219]: apparently there was another error. going to figure it out later

oh oop i didn't see your comment

**Pull request merged**